### PR TITLE
wps.xml - wps security

### PR DESCRIPTION
**Pull request by danfruehauf**
allow just one job sync and one job async

Will have to be configured from chef, but that's a good start too

### CODE REVIEW

**Comment by julian1** [comment]
Is it not possible to target the specific wps job?

**Comment by danfruehauf** [comment]
I'm working on that to be honest. I didn't quite succeed to limit that specific job - but it should be possible IMO.

**Comment by julian1** [comment]
Are you making any progress on this, or do you want me to merge this?

**Comment by danfruehauf** [comment]
I'll make more progress - I'm just not sure it'll be possible to limit just the job. Still working on it.

**Comment by danfruehauf** [comment]
GTG.

File: wps.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<wps>
+  <id>WPSInfoImpl--6b136ac:14d99e67450:-7ffd</id>
+  <enabled>false</enabled>
```
Comment (danfruehauf) [inline wps.xml]: @julian1 notice this is disabled by default, will be enabled by chef if necessary.

**Pull request merged**